alert-policies/amazon-workspaces/ConnectionFailure.yml

```diff
@@ -0,0 +1,27 @@
+name: High Connection Failure
+
+description: |+
+  This alert is triggered when over 100 Connection Failure bounce in a 10 minute period.
+
+type: STATIC
+nrql:
+  query: "SELECT count(`aws.workspaces.ConnectionFailure`) as 'Query' FROM Metric"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 100
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 600
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
```

Comment:
@rahul188 Just suggestion if we have to show the warnings before critical. 
For your reference please check this below link:
https://github.com/newrelic/newrelic-quickstarts/blob/main/alert-policies/apache-zooKeeper/RejectedConnection.yml